package.json

```diff
@@ -2,13 +2,14 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "pulp build --censor-lib --strict",
+    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
```

Comment:
There's no JS in this library? Plus eslint has replaced both of these tools now.